content/blog/pulumi-with-aiven-for-influxdb/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Zero Downtime InfluxDB Migration with Pulumi and Aiven"
+date: 2021-02-18
+meta_desc: "Zero downtime Pulumi in Aiven when transferring InfluxDB from cloud to cloud."
+meta_image: aiven.png
+authors:
+    - trevor-kennedy
+tags:
+    - Aiven
+    - InfluxDB
+---
+
```

Comment:
@spara - can we add a CTA here that points to our webinar? 
"Join Pulumi and Aiven experts on Wednesday, March 17th to learn more about delivering OSS databases with zero downtime" https://www.pulumi.com/resources/multicloud-oss-database-deployments-with-aiven/

Reply:
CTA added above the cut line